ec-europa/eubfr-data-lake
# feat(acl): allow only users from specific groups - EUBFR-36

The user must use AWS Signature (Postman can do that) and be a member of the "Producers" group in AWS IAM in order to request a presigned url.

Test scenario:
- first make sure your IAM user belongs to the "Producers" group
- then open Postman and create a new GET request to your `storage/signed_url` endpoint with `AWS Signature` as authorization type. Fill in your API keys. Add the `x-amz-meta-producer-key` header with `whateveryouwant.csv` as a value (should be a filename)
- send the request, it should return a new URL
- click on the url, it opens a new request tab in postman
- change the method to PUT and the body to `binary`. Choose a file to upload and send the request.
- now check the AWS environment (S3 bucket, dynamoDBs)
